platform-sdk/docs/proposals/TSS-Library/TSS-Library.md

```diff
@@ -223,29 +232,28 @@ Given the input:
 
 1. Serialize statement
 2. Initialize a pseudorandom number generator using the `SHA256` hash of (1)
-3. `x` = Random out of the random generator in (2)  
+3. `x` = Random out of the random generator in (2)
 4. Generate random values `α`, `ρ` from the `SignatureScheme.field` out of rng
 5. `g` = `SignatureScheme.publicKeyGroup.generator`
-  5.1    `F = g^ρ`
-  5.2    `A = g^α`
+   5.1    `F = g^ρ`
+   5.2    `A = g^α`
 6. Calculate `Y` by transforming elements of the statement (public keys, sids) multiplied by `ρ`, then adding `A`
-    
 7. Serialize (`x`, `F`, `A`, `Y`)
 8. Recompute the pseudorandom number generator seed using the hash of (7)
 9. `x'` = Random out of the random generator in (8)
-    
 10. `z_r` = `x'` * `witness.randomness` + `ρ`
 11. `z_a` = `x'` + `α` + Sum(`sᵢ` * `x`.pow(`sidᵢ`))
-    
-Return the proof composed of `F`, `A`, `Y`, `z_r`, and `z_a`
 
+Return the proof composed of `F`, `A`, `Y`, `z_r`, and `z_a`
 
 ##### Outside of scope
+
 Using an established channel, each participant will broadcast a single message to be received by all participants
 while waiting to receive other participants' messages. This functionality is critical for the protocol to work but needs to be handled outside the library.
 Each participant will validate the received message against the commitment and the NIZKs proof. Invalid messages need to be discarded.
 
 ##### Validation of TssMessage
+
```

Comment:
Is the addition of all these extra lines after headings intentional, or did Spotless do that?
(If spotless did this, then we need to fix spotless; this just looks terrible and is opposite to what every file we have does currently)